cirq/contrib/noise_models/noise_models.py

```diff
@@ -321,11 +321,11 @@ def simple_noise_from_calibration_metrics(calibration: engine.Calibration,
     readout_error_probs: Dict['cirq.Qid', float] = {}
 
     if depol_noise:
-        # TODO: replace with Pauli error once it's available.
-        # Github issue: https://github.com/quantumlib/Cirq/issues/2832
+        # In the single-qubit case, Pauli error and the depolarization fidelity
+        # differ by a factor of 4/3.
```

Comment:
Isn't this exactly the factor that average error and pauli error differ by?  Should we use average error instead?

I think you might want to use that rather than introduce a constant factor all over.

Reply:
Using the recent metrics as an example, we have:
```
# 1-qubit metrics for 5_2
avg. error:   0.0014671
pauli error:  0.0022007
```
It doesn't look like using average error would be equivalent to `pauli_err * 4 / 3` here.

Reply:
You are correct.  I took a look again at the supremacy supplement and depolarization error and average error are separately calculated.